apps/src/blockly/addons/shortcut.js

```diff
@@ -1,196 +0,0 @@
-import GoogleBlockly from 'blockly/core';
-
-/**
- * Adds a keyboard shortcut that will store copy information for a block
- * in localStorage. After switching to v7, we can replace this file with:
- * https://github.com/google/blockly-samples/tree/master/plugins/cross-tab-copy-paste
- */
-const blockCopyToStorageShortcut = function() {
-  const copyShortcut = {
-    name: 'copy',
-    preconditionFn: function(workspace) {
-      return (
-        !workspace.options.readOnly &&
-        !GoogleBlockly.Gesture.inProgress() &&
-        Blockly.selected &&
-        Blockly.selected.isDeletable() &&
-        Blockly.selected.isMovable()
```

Comment:
Note that this removal is fine, because the plugin already does this by default: https://github.com/google/blockly-samples/blob/304a875648bac4c7d78747a298dc8b2a1d28ba5b/plugins/cross-tab-copy-paste/src/index.js#L184-L185